bin/openshift-operations

```diff
@@ -12,5 +12,10 @@ TopologicalInventoryApiClient.configure do |config|
   config.host = "#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"]}:#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"]}"
 end
 
+SourcesApiClient.configure do |config|
+  config.scheme = "#{ENV["SOURCES_SCHEME"]}"
```

Comment:
I think the env value is already a string so the interpolation is unnecessary.

Reply:
:+1: